python/lsst/dax/imgserv/cli_v1.py

```diff
@@ -42,8 +42,24 @@
 from lsst.dax.imgserv.hashutil import Hasher
 from lsst.dax.imgserv.jsonutil import flatten_json
 
+try:
+    from ConfigParser import RawConfigParser, NoSectionError
+except ImportError:
+    from configparser import RawConfigParser, NoSectionError
```

Comment:
What's with this?  Seems like a capitalization difference?  Is this across versions?

Reply:
This is python2 compatibility shim and should be removed.